### DISCUSSION TITLE
Port Forwarding like Docker Desktop for nerdctl run containers?

### DISCUSSION THREAD

**Discussion by jonstelly** [rancher-sandbox/rancher-desktop#979] [Q&A] [answered]
I'd like to get rid of Docker Desktop completely and switch over to Rancher Desktop.  I see the Port Forwarding page/settings that looks like it will port forward for any kubernetes services but I'd like to be able to port forward for containers I run in containerd but outside of kubernetes like:

```
nerdctl run -it --rm -e "ACCEPT_EULA=Y" -e "SA_PASSWORD=???" -p 1433:1433 mcr.microsoft.com/mssql/server
```

I'd like to have `localhost:1433` from Windows be forwarded to this container.  Is that something that's already supported or do I just need to run MSSQL in Kubernetes for the easiest option?

**Comment by AkihiroSuda** [comment, marked as answer]
This should already work (on macOS host, at least)

**Reply by jonstelly** [reply]
Thanks, I'll test this again.  It's Windows and didn't work for localhost when I tried so I had to roll back to Docker Desktop.  I'll get Rancher Desktop reinstalled, try again and submit an issue if it's not something like a firewall issue.

**Reply by jandubois** [reply]
> It's Windows and didn't work for localhost

@jonstelly I think on Windows `localhost` only has an IPv6 address, and you need to use `127.0.0.1` instead.
